resources/js/src/app/directives/navigation/renderCategory.js

```diff
@@ -37,13 +37,14 @@ Vue.directive("render-category",
                     }
                 }
                 // check if user click the opened category and change the ui handling
-                else if (openCategory && openCategory.contains(event.target))
-                {
-                    store.dispatch("selectCategory", {categoryId: parseInt(el.dataset.categoryId)});
-                }
-                else if (document.body.classList.contains("no-touch") || binding.value.alwaysOpen)
+                else if ((openCategory && openCategory.contains(event.target)) || document.body.classList.contains("no-touch") || binding.value.alwaysOpen)
                 {
                     store.dispatch("selectCategory", {categoryId: parseInt(el.dataset.categoryId)});
+
+                    if (binding.value.scrollToTop)
+                    {
+                        $("html, body").animate({scrollTop: 0}, "slow");
```

Comment:
Ich würde in `scrollToTop` einen Integer binden, der die Geschwindigkeit der Animation angibt. So können Theme-Entwickler darauf leichter Einfluss nehmen. Die Logik wäre dann:
```
if (!isNullOrUndefined(binding.value.scrollToTop))
{
    $("html, body").animate({scrollTop: 0}, binding.value.scrollToTop);
}
```